src/platforms/esp32/components/avm_builtins/network_driver.c

```diff
@@ -541,15 +541,16 @@ static void send_ap_sta_disconnected(ClientData *data, uint8_t *mac)
     send_atom_mac(data, AP_STA_DISCONNECTED_ATOM, mac);
 }
 
-static void send_ap_sta_ip_acquired(ClientData *data, ip4_addr_t *ip)
-{
-    TRACE("Sending ap_sta_ip_acquired back to AtomVM\n");
-    Context *ctx = data->ctx;
-    port_ensure_available(ctx, ((4 + 1) + (2 + 1)) * 2);
-    term ip_term = tuple_from_addr(ctx, ntohl(ip->addr));
-    term reply = port_create_tuple2(ctx, AP_STA_IP_ASSIGNED_ATOM, ip_term);
-    send_term(data, reply);
-}
+// TODO Enable when we move to IDF-SDK v3.3 or later
+// static void send_ap_sta_ip_acquired(ClientData *data, ip4_addr_t *ip)
+// {
+//     TRACE("Sending ap_sta_ip_acquired back to AtomVM\n");
+//     Context *ctx = data->ctx;
+//     port_ensure_available(ctx, ((4 + 1) + (2 + 1)) * 2);
+//     term ip_term = tuple_from_addr(ctx, ntohl(ip->addr));
+//     term reply = port_create_tuple2(ctx, AP_STA_IP_ASSIGNED_ATOM, ip_term);
+//     send_term(data, reply);
+// }
```

Comment:
why are we keeping this function?

Reply:
For a future enhancement to the network driver to support callbacks when a station is assigned an IP address (when the device is in AP mode).  This was not supported in 3.x, but is in 4.x.  This entire driver needs to be re-written for 4.x and beyond, once we drop support for 3.  But not in this PR.

Reply:
removed